builder.Makefile

```diff
@@ -47,6 +47,7 @@ compose-plugin:
 .PHONY: cross
 cross:
 	GOOS=linux   GOARCH=amd64 $(GO_BUILD) $(TAGS) -o $(COMPOSE_BINARY)-linux-x86_64 ./cmd
+	GOOS=linux   GOARCH=ppc64le $(GO_BUILD) $(TAGS) -o $(COMPOSE_BINARY)-linux-ppc64le ./cmd
```

Comment:
is `ppc64le` the output for `uname -m` on PowerPC 64?

Reply:
yes @ndeloof